### DISCUSSION TITLE
Haptic feedback on Samsung phone

### DISCUSSION THREAD

**Discussion by sybenx** [rafaelvcaetano/melonDS-android#739] [General]
After updating from beta 1.5 to beta 1.6, the haptic vibration got much worse. Before, vibration was using the same feedback as pressing virtual keyboard buttons, after this update the vibration is not "clicky" anymore.

**Comment by rafaelvcaetano** [comment]
The haptic feedback was changed to allow more customization and by default, it's much stronger than before. You should be able to adjust the strength of the vibration to what you prefer in Settings -> Input -> Vibration strength

**Reply by sybenx** [reply]
i did notice i can make it stronger or weaker in the settings but it's not the same 'kind' of vibration. it's a buzz instead of a bump. the strongest setting is closer but not 'haptic feedback', it's a long vibration. and imo much worse. this is true on both samsung s10e and samsung note9.

**Reply by rafaelvcaetano** [reply]
OK, I see what you mean. I have a Galaxy S10 and can definitely see the difference you are talking about. I'll try to find a way to bring the old feeling again while maintaining the customization.
